### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by ErenKaya31** [DEMOLITIONDON96/Demolition-Engine#8] [General (Friday Night Chattin')]
hi there nice engine better then those fucking vdab editions

**Comment by ThatOneSillyGuyDon** [comment]
T h a n k .

**Reply by ErenKaya31** [reply]
why do you speak like that is that a alien ang or something?

**Reply by ThatOneSillyGuyDon** [reply]
> why do you speak like that is that a alien ang or something?

Uhhh, ummm, no, uhh, I’m not an alien, uhhh, aummmm, *bans you from the repo*
